Sources/EdgeConstants.swift

```diff
@@ -99,6 +103,8 @@ enum EdgeConstants {
         static let EVENT_ID = "_id"
         static let META = "meta"
         static let IMPLEMENTATION_DETAILS = "implementationDetails"
+        static let REQUEST = "request"
+        static let PATH = "path"
 
```

Comment:
These JSONKeys are used mainly for the network request body, let's keep just the EventDataKeys ones.